app/controllers/discover_controller.rb

```diff
@@ -1,3 +1,5 @@
 class DiscoverController < ApplicationController
-  def index; end
+  def index
```

Comment:
I'm actually using a `require_login` method in our `ApplicationController` so this can be changed to a `before_action :require_login` on line 2. We should change it later though.